README.md

```diff
@@ -1,5 +1,8 @@
 # MMM-JsonTable
 
+All credit to @timdows her.
```

Comment:
Credits are good and would be appropriate for a fork. But here I don't think it should be listed.

Reply:
Sorry never ment readme for PR